src/UI/Button.elm

```diff
@@ -145,6 +152,7 @@ defaultOptions : Options
 defaultOptions =
     { width = WidthShrink
     , size = Size.default
+    , id = ""
```

Comment:
I think a Maybe would be better. Otherwise we'll have lots of empty strings in the model and I'm not sure how much overhead that will generate for the JS VM.

Reply:
the overhead is lesser for the empty string than it is for the maybes, but the id should be optional and the best way to represent that is using a maybe